app/main.py

```diff
@@ -30,3 +33,8 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/eventedit")
+def eventedit(request: Request):
```

Comment:
Maybe better to place in a `routes/evnets.py` file.

Reply:
This doesn't seem to be correct semantic usage of a get request and should instead be: @app.put("/event").